lib/rollbar/delay/resque.rb

```diff
@@ -23,7 +23,12 @@ def self.perform(payload)
         end
 
         def perform(payload)
-          Rollbar.process_payload_safely(payload)
+          begin
+            Rollbar.process_payload_safely(payload)
+          rescue
+            # Raise the exception so Resque can track the errored job
```

Comment:
I wanted to be explicit here for the reraise. I can remove it and change the comment. Better for you?